src/components/share-structure/ShareStructure.vue

```diff
@@ -87,7 +87,8 @@
             ]"
           >
             {{ row.item.name }}
-            <ActionChip v-if="row.item.action && isEditMode" :actionable-item="row.item" :edited-label="editedLabel" />
+            <ActionChip v-if="row.item.action && isEditMode" :actionable-item="row.item" :edited-label="editedLabel"
+            class="pb-2" />
```

Comment:
Did you consider adding this to the SCSS at the bottom of this file instead?

I haven't played with this recently but maybe here?
https://github.com/bcgov/bcrs-shared-components/blob/fcc2b603547d9cdba4d6fbd2e621b98846a4d247/src/components/share-structure/ShareStructure.vue#L1031
or here?

The advantage of the SCSS approach is that it targets "all but the last" element.
https://github.com/bcgov/bcrs-shared-components/blob/fcc2b603547d9cdba4d6fbd2e621b98846a4d247/src/components/share-structure/ShareStructure.vue#L1060

Reply:
What would the advantage be of targeting all but the last element?

Reply:
Because you only want a dashed between element, like this:
A
---
B
---
C

And not after element, like this:
A
---
B
---
C
---

Reply:
Okay I see what you are saying. From the design there are dashed lines at the end of the series. 
![5382642e-3e73-46ce-98d9-fbd2829df351](https://user-images.githubusercontent.com/60455392/189768048-e22ae877-4000-425b-a7ab-4ffb6d567976.png)

Reply:
Unless the design is different from the current code, those are not dashed lines at the end of the series -- they are dashed lines **between shares**. Theoretically, there is nothing "after" classes/series.

Reply:
Oh I see what you are saying. The action chip is for every the blue added sign so I do want it after every series/classes

Reply:
Could you please show some examples with mutliple series and classes, some with the chip and some without? (To verify the added whitespace after the end of each series and class.)

Reply:
And please verify the whitespace when the names wrap (which you can prob do by making your window narrower).